### PR TITLE
Wallchart snap

### PR DESCRIPTION
**Pull request by nicl**
I need to review the code here myself too. Snaps are not a great abstraction at the moment because our CSS/JS are global so they cannot be ignorant of their context.

Normally for football snaps we add them to the Admin app. In this case I'll manually construct the snap URL but may add a separate PR for the admin UI later if considered necessary.

## What does this change?

Adds a spider snap to use on fronts during the world cup.

## Screenshots

screen shot 2018-07-02 at 14 23 25

## What is the value of this and can you measure success?

It is considered crucial for our world cup support. At the moment we have a hacky solution of screenshots, which is annoying for the editorial team to have to maintain.

## Tested

Locally only so far (waiting for CODE).

### CODE REVIEW

File: static/src/stylesheets/head.footballSnaps.scss [hunk 1]
```diff
@@ -7,5 +7,6 @@
 @import 'module/football/_match-stat';
 @import 'module/football/_match-summary';
 @import 'module/football/_matches';
+/*@import 'module/survey/_dropdown';*/
```
Comment (nicl) [inline static/src/stylesheets/head.footballSnaps.scss]: For some reason I can't import this - perhaps it's not a proper SCSS module? In any case, it seems to get included automatically.

I'll remove but wanted to ask about this before doing so.

Comment (NataliaLKB) [reply static/src/stylesheets/head.footballSnaps.scss]: If it is already included in the page can we delete this commented out one?

**Review by NataliaLKB — APPROVED** [review]
Lgtm 👍 . One comment, but otherwise good to go.

**Comment by nicl** [comment]
This is very difficult to test as requires fronts tool code and a deploy to code but I am fairly sure it works. It's difficult to be confident I haven't broken styling of some of the other football snaps/charts although I've done my best to look.

I'll keep a close eye and revert if it breaks things though.

**Pull request merged**